src/www/ui/admin-license-file.php

```diff
@@ -202,64 +205,52 @@ function LicenseList($namestr, $filter)
-    $text = _("Active");
-    $ob .= "<th>$text</th>";
-    $text = _("SPDX Compatible");
-    $ob .= "<th>$text</th>";
-    $text = _("Shortname");
-    $ob .= "<th>$text</th>";
-    $text = _("Fullname");
-    $ob .= "<th>$text</th>";
-    $text = _("Text");
-    $ob .= "<th>$text</th>";
-    $text = _("URL");
-    $ob .= "<th>$text</th>";
-    $ob .= "</tr>";
-    $lineno = 0;
+    $ob .= "<table id='adminLicenseTable' rules='rows' width='100%' cellpadding='3'>";
+    $ob .= "<thead>
+              <tr>
+                <th>Edit</th>
+                <th>Checked</th>
+                <th>SPDX Compatible</th>
+                <th>Shortname</th>
+                <th>Fullname</th>
+                <th>Text</th>
+                <th>URL</th>
+              </tr>
+           </thead>";
+    $ob .= "</table>";
+    $tableBodyData = [];
+
     while ($row = pg_fetch_assoc($result))
     {
-      if ($lineno++ % 2)
-        $style = "style='background-color:lavender'";
-      else
-        $style = "";
-      $ob .= "<tr $style>";
+      $marydone = ($row['marydone'] == 't') ? "Yes" : "No";
 
-      // Edit button brings up full screen edit of all license_ref fields
-      $ob .= "<td align=center><a href='";
-      $ob .= Traceback_uri();
-      $ob .= "?mod=" . $this->Name .
-           "&rf_pk=$row[rf_pk]".
-           "&req_marydone=$_REQUEST[req_marydone]&req_shortname=$_REQUEST[req_shortname]' >".
-           "<img border=0 src='" . Traceback_uri() . "images/button_edit.png'></a></td>";
 
-      $marydone = ($row['marydone'] == 't') ? "Yes" : "No";
-      $text = _("$marydone");
-      $ob .= "<td align=center>$text</td>";
-      $rf_active = ($row['rf_active'] == 't') ? "Yes" : "No";
-      $text = _("$rf_active");
-      $ob .= "<td align=center>$text</td>";
       $rf_spdx_compatible = ($row['rf_spdx_compatible'] == 't') ? "Yes" : "No";
-      $text = _("$rf_spdx_compatible");
-      $ob .= "<td align=center>$text</td>";
-      $ob .= "<td align=center>$row[rf_shortname]</td>";
-      $ob .= "<td align=left>$row[rf_fullname]</td>";
-      $vetext = htmlspecialchars($row['rf_text']);
-      $ob .= "<td><textarea readonly=readonly rows=3 cols=40>$vetext</textarea></td> ";
-      $ob .= "<td align=left>$row[rf_url]</td>";
-      $ob .= "</tr>";
+      $vetext = $row['rf_text'];
+      $row = array(
+        "<td align=center><a href='".Traceback_uri() . "?mod=" . $this->Name .
+          "&rf_pk=$row[rf_pk]&req_marydone=$_REQUEST[req_marydone]&req_shortname=$_REQUEST[req_shortname]' >".
+          "<img border=0 src='" . Traceback_uri() . "images/button_edit.png'></a></td>",
+        $marydone,
+        $rf_spdx_compatible,
+        "$row[rf_shortname]",
+        "$row[rf_fullname]",
+        "<textarea readonly rows='3' cols='40'>$vetext</textarea>",
+        "$row[rf_url]"
+      );
+
+      $tableBodyData[] = $row;
     }
+
+    $ob .= "
+    <script>
+     var licenseTable = '" . urlencode(json_encode(($tableBodyData))) . "';
```

Comment:
The $(document).ready()-pattern isn't working in that case. Also none of the other later loading functions work